src/legacy/server/logging/rotate/index.ts

```diff
@@ -30,12 +29,6 @@ export async function setupLoggingRotate(server: Server, config: KibanaConfig) {
     return;
   }
 
-  // We just want to start the logging rotate service once
-  // and we choose to use the master (prod) or the worker server (dev)
-  if (!isMaster && isWorker && process.env.kbnWorkerType !== 'server') {
```

Comment:
In master all workers are of type `server`, so this condition will always evaluate to false